### ISSUE TITLE
option to eliminate args from class signature

### ISSUE THREAD

**Issue by rdzman** [sphinx-contrib/matlabdomain#182]
I would find it useful to have an option to remove the function-style display of the class signature, with the parentheses and constructor argument list.

See [this example](https://github.com/sphinx-contrib/matlabdomain/pull/181#issuecomment-1516855465).

**Comment by joeced** [comment]
Agree. That is a good idea. I never thought about it really, but now it stands out in all my documentation.
